EasyPost/Exception.cs

```diff
@@ -52,4 +52,12 @@ public class ResourceAlreadyCreated : Exception
     public class ClientNotConfigured : Exception
     {
     }
+
+    [Serializable]
+    public class ResourceNotCreated : Exception
```

Comment:
Is this being used in other places? Correct me if I am wrong.

Reply:
Yeah, let's remove this bit if it's not in use.